spring-ai-openai/src/main/java/org/springframework/ai/openai/api/OpenAiApi.java

```diff
@@ -51,7 +52,7 @@ public class OpenAiApi {
 
 	private static final String DEFAULT_BASE_URL = "https://api.openai.com";
 	private static final String DEFAULT_EMBEDDING_MODEL = "text-embedding-ada-002";
-	private static final String SSE_DONE = "[DONE]";
+	private static final Predicate<String> SSE_DONE_PREDICATE = "[DONE]"::equals;
```

Comment:
I presume that the Predicate is just a code style improvement, not due to issues with the existing conditions?

Reply:
The documentation at https://platform.openai.com/docs/api-reference/chat/create explicitly states that it ends with '[DONE]', so in my opinion, using 'equals' would be clearer than 'contains.